### PR TITLE
Use seed-list name

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

File: modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala [hunk 1]
```diff
@@ -229,11 +229,11 @@ sealed abstract class Joining[F[_]: Async: GenUUID: SecurityProvider: KryoSerial
...
     } yield peer
 
-  private def validateWhitelisting(peer: PeerToJoin): F[Unit] =
-    whitelisting match {
+  private def validateSeedlist(peer: PeerToJoin): F[Unit] =
+    seedlist match {
       case None => Applicative[F].unit
       case Some(entries) =>
-        if (entries.contains(peer.id)) Applicative[F].unit else PeerNotWhitelisted(peer.id).raiseError[F, Unit]
+        if (entries.contains(peer.id)) Applicative[F].unit else PeerNotInSidelist(peer.id).raiseError[F, Unit]
```
Comment (TheMMaciek) [inline modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala:236]: PeerNotInSeedList

Comment (marcinwadon) [reply modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala:236]: `Seedlist` is used like `Whitelist`

File: modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/gossip/RumorValidator.scala [hunk 2]
```diff
@@ -69,23 +69,23 @@ object RumorValidator {
...
+      }.map(SignersNotInSidelist).toInvalidNec(signedRumor)
 
   }
 
   @derive(eqv, show)
   sealed trait RumorValidationError
   case class InvalidHash(calculatedHash: Hash, receivedHash: Hash) extends RumorValidationError
   case class InvalidSigned(error: SignedValidationError) extends RumorValidationError
-  case class SignersNotWhitelisted(signers: NonEmptySet[Id]) extends RumorValidationError
+  case class SignersNotInSidelist(signers: NonEmptySet[Id]) extends RumorValidationError
```
Comment (TheMMaciek) [inline modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/gossip/RumorValidator.scala:88]: `SignersNotInSeedList`

File: modules/shared/src/main/scala/org/tessellation/schema/cluster.scala [hunk 3]
```diff
@@ -46,7 +46,7 @@ object cluster {
   case class NodeStateDoesNotAllowForJoining(nodeState: NodeState) extends NoStackTrace
   case class PeerIdInUse(id: PeerId) extends NoStackTrace
   case class PeerHostPortInUse(host: Host, p2pPort: Port) extends NoStackTrace
-  case class PeerNotWhitelisted(id: PeerId) extends NoStackTrace
+  case class PeerNotInSidelist(id: PeerId) extends NoStackTrace
```
Comment (TheMMaciek) [inline modules/shared/src/main/scala/org/tessellation/schema/cluster.scala:49]: `PeerNotInSeedList`

File: modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala (hunk 1 above)
Comment (TheMMaciek) [reply modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/Joining.scala]: That's okay, but there is a typo as well.

**Review by TheMMaciek — APPROVED** [review]

**Pull request merged**